bib/internal/setup/setup.go

```diff
@@ -0,0 +1,81 @@
+package setup
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/osbuild/bootc-image-builder/bib/internal/podmanutil"
+	"github.com/osbuild/bootc-image-builder/bib/internal/utils"
+	"golang.org/x/sys/unix"
+)
+
+// EnsureEnvironment mutates external filesystem state as necessary
+// to run in a container environment.  This function is idempotent.
+func EnsureEnvironment() error {
+	osbuildPath := "/usr/bin/osbuild"
+	if utils.IsMountpoint(osbuildPath) {
```

Comment:
(nitpick^3) I feel a bit bad for mentioning it and ignoring is fine - `utils` vs `util` (plural/singular) similar to podmanutil here :) Feel free to ignore though if it's too much nitpicking (

Reply:
OK I renamed